.github/workflows/ci.yml

```diff
@@ -5,12 +5,11 @@ on:
     branches:
       - main
     tags:
-      - '**'
+      - "**"
   pull_request: {}
 
 env:
   COLUMNS: 150
-  PDM_DEPS: 'urllib3<2,git+https://github.com/adriangb/findpython.git@version-timeout-env'
```

Comment:
pdm 2.9.3  up the min version, so delete this

Reply:
Didn't know this. Thanks!